.circleci/config.yml

```diff
@@ -79,11 +79,10 @@ jobs:
           POSTGRES_USER: form-builder
           POSTGRES_DB: form-builder
       - image: redis
-      - image: localstack/localstack:0.11.2
+      - image: localstack/localstack:latest
```

Comment:
using `latest` is the only way to get the fix. Need a follow on change to pin to a version once 3.0.0 drops.